.github/workflows/tests.yml

```
@@ -0,0 +1,48 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+
+  workflow_dispatch:
+
+  schedule:
+    # https://crontab.guru/#15_14_*_*_*
+    - cron: 15 14 * * *
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Cache Pip
+        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        with:
+          path: ~/.cache/pip
+          # invalidate the cache anytime a requirements.txt changes
+          key: ${{ github.workflow }}-${{ hashFiles('**/requirements.txt') }}
+
+      - name: Setup Python
+        uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
+        with:
+          python-version: '3.11'
+
+      - name: Install Dependencies
+        run: pip install --quiet -r requirements.txt -r combine-durations/requirements.txt -r template-files/requirements.txt
```

```suggestion
        run: pip install --quiet -r requirements.txt -r template-files/requirements.txt
```

I can't find `combine-durations/requirements.txt` in the PR, did you forget to add it or is the call here old?

---

Ah, thanks! I've had tomatoes on my eyes, as the German saying goes